test.rb

```diff
@@ -0,0 +1,36 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+def a
+  def b
+    1
+    ['a','c']
+    'C'
+    "A"
+    ?a
+  end
+end
+
+aaa;
+
+not 1
+
+ l = lambda {|a| a }
+ l.(x)
+l[x]
+l.call(x)
+
+  def b(x)
+    
```

Comment:
__[development][rubocop]__
* Method `Object#b` is defined at both /sideci/sandbox2/test.rb:9 and /sideci/sandbox2/test.rb:27.
* Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->